emain/emain-window.ts

```diff
@@ -699,11 +701,16 @@ ipcMain.on("delete-workspace", (event, workspaceId) => {
             console.log("user cancelled workspace delete", workspaceId, ww?.waveWindowId);
```

Comment:
It won't let me add a comment above this line, but you should also remove lines 690-692 and edit the message in the Delete Workspace message box to remove the part mentioning that the window will be closed, since we're changing this behavior.